.travis.yml

```diff
@@ -1,8 +1,26 @@
 language: rust
+
 rust:
   - stable
   - beta
   - nightly
+  - nightly-2017-10-08
+
 matrix:
   allow_failures:
     - rust: nightly
+
+env:
+  global:
+    - CLIPPY_VERSION=0.0.165
+
+before_script:
+  - bash -c 'if [[ "$TRAVIS_RUST_VERSION" == "nightly-2017-10-08" ]]; then
```

Comment:
Why does this need to be tied to this particular nightly release?

Reply:
Because older clippy versions regularly break with newer nightly. This just ensures we install a working (clippy, nightly) tuple. The specific date for nightly doesn't matter as long as it is compatible with clippy version.

Reply:
Ah, that's right.

Reply:
For the sake of completeness, 2017-10-08 nightly does not have any peculiar property. It is just an arbitrary date which I tested to be compatible with clippy 0.0.165.